AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/ModelValueConverter+SQLite.swift

```diff
@@ -43,7 +43,11 @@ public struct SQLiteModelValueConverter: ModelValueConverter {
         case .enum:
             return (value as? EnumPersistable)?.rawValue
         case .model:
-            return (value as? Model)?.id
+            if let modelType = (value as? Model),
+               let modelSchema = ModelRegistry.modelSchema(from: modelType.modelName) {
+                return modelType.identifier(schema: modelSchema).stringValue
```

Comment:
maybe `modelType` here can just be called `modelInstance` or `model`? Could this code fail for Flutter use cases? ie. what if the instance doesn't contain `modelName`?

Reply:
> maybe modelType here can just be called modelInstance or model?
Makes sense, changing to `modelInstance`.

> Could this code fail for Flutter use cases? ie. what if the instance doesn't contain modelName?
Flutter always provides a valid `modelName` and populates the schema accordingly (checked with @HuiSF)